disnake/message.py

```diff
@@ -660,6 +660,8 @@ class InteractionReference:
         The type of interaction.
     name: :class:`str`
         The name of the application command.
+        This also includes group and subcommand name if applicable (separated by spaces)
+        for interactions created after July 18th, 2022.
```

Comment:
Possible rewording:
```suggestion
        The name of the application command, including group and subcommand name if applicable
        (separated by spaces).

        .. note::
            
            For interaction references created before July 18th, 2022, this will not include group or subcommand names.
```